images/router/haproxy/conf/haproxy-config.template

```diff
@@ -33,6 +33,11 @@
 
 global
   maxconn {{env "ROUTER_MAX_CONNECTIONS" "20000"}}
+{{- if isTrue (env "ROUTER_ENABLE_TRHEADS") }}
+  nbthread {{env "ROUTER_THREADS" "4"}}
```

Comment:
Let's just do it based on whether ROUTER_THREADS is defined and > 1, and then default to 1 thread.

Reply:
+1, ROUTER_ENABLE_TRHEADS seems redundant.